charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -37,17 +37,22 @@ containers:
           fieldRef:
             apiVersion: v1
             fieldPath: status.podIP
+      - name: KUBE_NODE_NAME
+        valueFrom:
+          fieldRef:
+            apiVersion: v1
+            fieldPath: spec.nodeName
       {{- with .Values.extraEnvs }}
       {{- . | toYaml | nindent 6 }}
       {{- end }}
-    livenessProbe:
-      httpGet:
-        path: /
-        port: 13133
-    readinessProbe:
-      httpGet:
-        path: /
-        port: 13133
+    # livenessProbe: # TODO: does not work. it keeps killing pod. 
+    #   httpGet:
+    #     path: /
+    #     port: 13133
+    # readinessProbe:
+    #   httpGet:
+    #     path: /
+    #     port: 13133
```

Comment:
This works prior to this commit, do you know what might have affected it? Would be nice to sort out before merging.

Reply:
Hmm I just tested it and it is working again. I will uncomment this part. Thanks!